### PR TITLE
fix: thanos receive ingress path

### PR DESCRIPTION
**Pull request by srodenhuis**
Fix for ingress configuration to enable remote-write to Thanos for external clusters

### CODE REVIEW

**Review by dennisvankekem — APPROVED** [review]
LGTM

File: core.yaml [hunk 1]
```diff
@@ -376,7 +376,7 @@ adminApps:
       - svc: thanos-receive
         port: 19291
         namespace: monitoring
-        paths: [/api/v1/receive/]
+        paths: [/api]
```
Comment (j-zimnowoda) [inline core.yaml:379]: According to Thanos docs the path should be `/api/v1/receive`. Could you tell us the reason of exposing whole thanos-receive API  ?
See: https://thanos.io/v0.28/components/receive.md/#example

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]

File: core.yaml (hunk 1 above)
Comment (srodenhuis) [reply core.yaml:379]: Setting path to /api/v1/receive in ingress and vs results in 404

Comment (j-zimnowoda) [reply core.yaml:379]: ok, then it sounds indeed that more is needed. Thanks

**Review by j-zimnowoda — APPROVED** [review]

**Pull request merged**